integ/package.json

```diff
@@ -71,38 +71,38 @@
     "ts-jest": "^26.4.4"
   },
   "dependencies": {
-    "@aws-cdk/aws-docdb": "1.75.0",
-    "@aws-cdk/aws-ec2": "1.75.0",
-    "@aws-cdk/aws-ecr": "1.75.0",
-    "@aws-cdk/aws-ecs": "1.75.0",
-    "@aws-cdk/aws-efs": "1.75.0",
-    "@aws-cdk/aws-elasticloadbalancingv2": "1.75.0",
-    "@aws-cdk/aws-iam": "1.75.0",
-    "@aws-cdk/aws-logs": "1.75.0",
-    "@aws-cdk/aws-route53": "1.75.0",
-    "@aws-cdk/aws-s3": "1.75.0",
-    "@aws-cdk/aws-s3-assets": "1.75.0",
-    "@aws-cdk/aws-secretsmanager": "1.75.0",
-    "@aws-cdk/core": "1.75.0",
+    "@aws-cdk/aws-docdb": "1.78.0",
+    "@aws-cdk/aws-ec2": "1.78.0",
+    "@aws-cdk/aws-ecr": "1.78.0",
+    "@aws-cdk/aws-ecs": "1.78.0",
+    "@aws-cdk/aws-efs": "1.78.0",
+    "@aws-cdk/aws-elasticloadbalancingv2": "1.78.0",
+    "@aws-cdk/aws-iam": "1.78.0",
+    "@aws-cdk/aws-logs": "1.78.0",
+    "@aws-cdk/aws-route53": "1.78.0",
+    "@aws-cdk/aws-s3": "1.78.0",
+    "@aws-cdk/aws-s3-assets": "1.78.0",
+    "@aws-cdk/aws-secretsmanager": "1.78.0",
+    "@aws-cdk/core": "1.78.0",
     "aws-rfdk": "0.21.0",
-    "aws-sdk": "^2.799.0"
+    "aws-sdk": "^2.810.0"
   },
   "peerDependencies": {
-    "@aws-cdk/aws-docdb": "1.75.0",
-    "@aws-cdk/aws-ec2": "1.75.0",
-    "@aws-cdk/aws-ecr": "1.75.0",
-    "@aws-cdk/aws-ecs": "1.75.0",
-    "@aws-cdk/aws-efs": "1.75.0",
-    "@aws-cdk/aws-elasticloadbalancingv2": "1.75.0",
-    "@aws-cdk/aws-iam": "1.75.0",
-    "@aws-cdk/aws-logs": "1.75.0",
-    "@aws-cdk/aws-route53": "1.75.0",
-    "@aws-cdk/aws-s3": "1.75.0",
-    "@aws-cdk/aws-s3-assets": "1.75.0",
-    "@aws-cdk/aws-secretsmanager": "1.75.0",
-    "@aws-cdk/core": "1.75.0",
+    "@aws-cdk/aws-docdb": "1.78.0",
+    "@aws-cdk/aws-ec2": "1.78.0",
+    "@aws-cdk/aws-ecr": "1.78.0",
+    "@aws-cdk/aws-ecs": "1.78.0",
+    "@aws-cdk/aws-efs": "1.78.0",
+    "@aws-cdk/aws-elasticloadbalancingv2": "1.78.0",
+    "@aws-cdk/aws-iam": "1.78.0",
+    "@aws-cdk/aws-logs": "1.78.0",
+    "@aws-cdk/aws-route53": "1.78.0",
+    "@aws-cdk/aws-s3": "1.78.0",
+    "@aws-cdk/aws-s3-assets": "1.78.0",
+    "@aws-cdk/aws-secretsmanager": "1.78.0",
+    "@aws-cdk/core": "1.78.0",
     "aws-rfdk": "0.21.0",
-    "aws-sdk": "^2.799.0"
```

Comment:
We didn't update versions for `aws-sdk`, `typescript`, and  `@types/node` in `aws-rfdk/lambda-layers/package.json`. Do we need to update them?

Reply:
Ah good catch, I missed this directory. I'll update this.